nncf/tensorflow/graph/metatypes/common.py

```diff
@@ -109,7 +109,9 @@
 
 LAYER_METATYPES_AGNOSTIC_TO_DATA_PRECISION_WITH_MULTIPLE_INPUTS = [
     layer_metatypes.TFConcatenateLayerMetatype,
-    op_metatypes.TFConcatOpMetatype
+    op_metatypes.TFConcatOpMetatype,
+    op_metatypes.TFMaximumOpMetatype,
+    op_metatypes.TFMinimumOpMetatype,
```

Comment:
That is correct. Simple option is to put `TFMaximumOpMetatype `and `TFMinimumOpMetatype` to the `LAYER_METATYPES_AGNOSTIC_TO_DATA_PRECISION_WITH_ONE_INPUT`. It will work, but actually, `maximum` and `minimum` has two inputs.

Reply:
Another option is to create separate groups for `concat` and `max/min` ops. What do you think?

Reply:
I would prefer to create a separate group.

Reply:
I created a separate group